### ISSUE TITLE
Incorrect written number for hundred numbers without trailing numbers in Spanish

### ISSUE THREAD

**Issue by pexposito** [yamadapc/js-written-number#72]
For:
```
writtenNumber(1101, { lang: "es" })
\\ it returns "mil ciento uno" -> CORRECT
```

for:
```
writtenNumber(1100, { lang: "es" })
\\ it returns "mil ciento" -> INCORRECT
\\ it should be "mil cien"
```

The solution is to modify the first unit for "es.json" language file to add "useBaseExceptionWhenNoTrailingNumbers" property:
```
{
      "singular": "ciento",
      "useBaseInstead": true,
      "useBaseException": [1],
      "useBaseExceptionWhenNoTrailingNumbers": true
}
```

**Comment by pexposito** [comment]
I've created a PR for this issue. Cheers!!!

**Comment by yamadapc** [comment]
Fixed as of v0.11.1, thanks
